modules/reliquary/components/DeligateClearButton.tsx

```diff
@@ -0,0 +1,47 @@
+import { HStack, Text } from '@chakra-ui/react';
+import React, { useEffect } from 'react';
+import { BeetsSubmitTransactionButton } from '~/components/button/BeetsSubmitTransactionButton';
+import { ToastType, useToast } from '~/components/toast/BeetsToast';
+import { useDelegateClear } from '../lib/useDelegateClear';
+import { useDelegation } from '../lib/useDelegation';
+
+interface Props {
+    size?: string;
+    rounded?: string;
+    w?: string;
+}
+
+export default function DeligateClearButton({ ...rest }: Props) {
```

Comment:
You can use 'ButtonProps' which is exported by chakra here

Reply:
Also nit: delegate is not spelt correctly in the components

Reply:
fixed both

Reply:
`ButtonProps` somehow meeses with `BeetsSubmitTransactionButtonProps` so reverted it